pkg/event/processor.go

```diff
@@ -242,7 +252,15 @@ func (p *QueueingEventProcessor) FlushEvents() {
 		}
 		if batchEventCount > 0 {
 			// TODO: figure out what to do with the error
```

Comment:
nit: This should go right before we call `DispatchEvent`

Reply:
or be greater than or equal which i prefer